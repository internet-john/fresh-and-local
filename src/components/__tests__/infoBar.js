import React from "react";
import renderer from "react-test-renderer";

import InfoBar from "../infoBar/infoBar";

describe("InfoBar", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<InfoBar season="Summer" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})