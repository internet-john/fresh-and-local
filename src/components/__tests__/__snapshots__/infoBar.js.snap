// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBar renders correctly 1`] = `
<div
  className="MuiFormGroup-root makeStyles-chipBar-1 MuiFormGroup-row"
>
  <label
    className="MuiFormControlLabel-root"
  >
    <span
      className="MuiSwitch-root"
    >
      <span
        aria-disabled={false}
        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-26 MuiSwitch-switchBase MuiSwitch-colorPrimary"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={null}
      >
        <span
          className="MuiIconButton-label"
        >
          <input
            className="PrivateSwitchBase-input-29 MuiSwitch-input"
            disabled={false}
            onChange={[Function]}
            type="checkbox"
            value="SHOW_FRUIT"
          />
          <span
            className="MuiSwitch-thumb"
          />
        </span>
      </span>
      <span
        className="MuiSwitch-track"
      />
    </span>
    <span
      className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
    >
      Fruits
    </span>
  </label>
  <label
    className="MuiFormControlLabel-root"
  >
    <span
      className="MuiSwitch-root"
    >
      <span
        aria-disabled={false}
        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-26 MuiSwitch-switchBase MuiSwitch-colorPrimary"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={null}
      >
        <span
          className="MuiIconButton-label"
        >
          <input
            className="PrivateSwitchBase-input-29 MuiSwitch-input"
            disabled={false}
            onChange={[Function]}
            type="checkbox"
            value="SHOW_VEG"
          />
          <span
            className="MuiSwitch-thumb"
          />
        </span>
      </span>
      <span
        className="MuiSwitch-track"
      />
    </span>
    <span
      className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
    >
      Vegetables
    </span>
  </label>
</div>
`;
